src/test/java/com/musseukpeople/woorimap/util/RepositoryTest.java

```diff
@@ -0,0 +1,11 @@
+package com.musseukpeople.woorimap.util;
+
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.context.annotation.Import;
+
+@DataJpaTest
+@Import(QueryDslTestConfig.class)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public abstract class RepositoryTest {
```

Comment:
어노테이션으로 바꿔도 될 것 같아요!
```suggestion
public @interface class RepositoryTest {
```

Reply:
확실히 어노태이션으로 하면 편하겠네요~?